dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.11
 
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y sqlite3
```

Comment:
I would not recommend you to use sqlite as primary storage. It's okay to store utilitary information in it, but in general avoid this DB.